dbsnp-importer/src/test/resources/application.properties

```diff
@@ -1,11 +1,10 @@
 spring.datasource.driver-class-name=org.hsqldb.jdbcDriver
-spring.datasource.url=jdbc:hsqldb:mem:db;DB_CLOSE_DELAY=-1
+spring.datasource.url=jdbc:hsqldb:mem:db;sql.syntax_pgs=true;DB_CLOSE_DELAY=-1
 spring.datasource.username=SA
 spring.datasource.password=
 
 # The same embedded database is reused by your whole test suite regardless of the number of application contexts.
 # Set to true to make sure that each context has a separate embedded database.
 spring.datasource.generate-unique-name=true
 
-spring.jpa.hibernate.ddl-auto=none
-sql.syntax_pgs=true
+spring.jpa.hibernate.ddl-auto=none
```

Comment:
This is minor but does having no newline cause issue with .properties files?

Reply:
Not that I know of, it has behaved correctly so far.